docs/multi-domains.md

```diff
@@ -0,0 +1,37 @@
+# Multi-domains
+
+ILC supports multi-domains, which means that you don't need to roll out personal instances of ILC for each of them, you just can use one instance for all of them.  
```

Comment:
```suggestion
ILC can handle requests coming from multiple domains, so that you don't need to roll out individual instances of the ILC for every domain, you just can use one instance for all of them.  
```